sklearn/feature_selection/_from_model.py

```diff
@@ -399,8 +420,24 @@ def partial_fit(self, X, y=None, **fit_params):
             The target values (integers that correspond to classes in
             classification, real numbers in regression).
 
-        **fit_params : dict
-            Other estimator specific parameters.
+        **partial_fit_params : dict
+            - If `enable_metadata_routing=False` (default):
+
+                Parameters directly passed to the `partial_fit` method of the
+                sub-estimator.
+
+            - If `enable_metadata_routing=True`:
+
+                Parameters passed to the `partial_fit` method of the
+                sub-estimator. They are ignored if `prefit=True`.
+
+                .. versionchanged:: 1.4
+                    `**partial_fit_params` are routed to the sub-estimator, if
+                    `enable_metadata_routing=True` is set via
+                    :func:`~sklearn.set_config`, which allows for aliasing.
```

Comment:
```suggestion
                    Only available if `enable_metadata_routing=True`,
                    which can be set by using
                    ``sklearn.set_config(enable_metadata_routing=True)``.
                    See :ref:`Metadata Routing User Guide <metadata_routing>` for
                    more details.
```

Reply:
Actually, I would suggest to leave this part below `.. versionchanged:: 1.4` mostly away. We already wrote all of this before and after. And it looks difficult to read.

What about this?

```
         **partial_fit_params : dict
            - If `enable_metadata_routing=False` (default):

                Parameters directly passed to the `partial_fit` method of the
                sub-estimator. They are ignored if `prefit=True`.

            - If `enable_metadata_routing=True`:

                Parameters safely routed to the `partial_fit` method of the
                sub-estimator. They are ignored if `prefit=True`.

                .. versionchanged:: 1.4
                    See :ref:`Metadata Routing User Guide <metadata_routing>` for
                    more details.
```

Reply:
I agree.